Add service labels to service.yaml helm template

Dentling opened the pull request:
#Closes https://github.com/comet-ml/opik/issues/3783

## Details
Currently it is not possible to add custom labels to the kubernetes service.
This PR introduces this feature.

## Change checklist
- [ ] User facing
- [ ] Documentation update

## Issues


## Testing

## Documentation

obezpalko reviewed (changes requested):
It will not work like that (no end for with)

 Labels should be added in helper

 If you want additional labels - it should be a dedicated field f.e. extraLabels

Dentling commented:
hi @obezpalko,

i fixed the syntax and added the "end"-block for the `with`.

I think this should not be a helper, since this is for labels that only the service should have.
I did it just like it has been done for the annotations below, which are also only for the Kubernetes-Service-Resource.
